features/step_definitions/aspects_steps.rb

```diff
@@ -88,24 +88,21 @@ def aspect_dropdown_visible?
   end
 end
 
-When /^I drag "([^"]*)" (up|down) (\d+) pixels?$/ do |aspect_name, direction, distance|
-  distance = distance.to_i * -1 if direction == "up"
-  page.execute_script %{
-    function drag() {
-      $("li.aspect:contains('#{aspect_name}')")
-        .simulate("drag-n-drop", { dy: #{distance}, interpolation: { stepWidth: 10, stepDelay: 5 } });
-    }
-    function loadScripts() {
-      $.getScript("/assets/jquery-simulate/jquery.simulate.js", function(){
-        $.getScript("/assets/jquery-simulate-ext/src/jquery.simulate.ext.js", function(){
-          $.getScript("/assets/jquery-simulate-ext/src/jquery.simulate.drag-n-drop.js", drag);
-        });
-      });
-    }
-    if (!$.simulate) { loadScripts(); } else { drag(); }
-  }
-  expect(find("#aspect_nav")).to have_css ".synced"
-end
+When /^I drag "([^"]*)" (up|down)$/ do |aspect_name, direction|
+  aspect_id = @me.aspects.where(name: aspect_name).first.id
+  aspect = find(:xpath, "//div[@id='aspect_nav']/ul/li[@data-aspect-id='#{aspect_id}']")
+  target = direction == "up" ? aspect.all(:xpath, "./preceding-sibling::li").last :
+                               aspect.all(:xpath, "./following-sibling::li").first
+  browser = aspect.base.driver.browser
+  mouse = browser.mouse
+  native_aspect = aspect.base.native
+  native_target = target.base.native
+  mouse.down native_aspect
+  mouse.move_to native_target, native_target.size.width/2, 0
+  sleep 1
+  mouse.up
+  expect(page).to have_no_css "#aspect_nav .ui-sortable.syncing"
+ end
```

Comment:
`end` at 105, 1 is not aligned with `When /^I drag "([^"]*)" (up|down)$/ do |aspect_name, direction|` at 91, 0